CONTRIBUTING.md

```diff
@@ -4,3 +4,8 @@
 
 :+1::tada: First off, I'm really glad you're reading this, because we need volunteer developers to help with the development of fAIr! :tada::+1:
 We welcome and encourage contributors of all skill levels and we are committed to making sure your participation in our tech collective is inclusive, enjoyable and rewarding. If you have never contributed to an open-source project before, we are a good place to start and will make sure you are supported every step of the way. If you have **any** questions, please ask!
+
+## :handshake: Thank you
+
+Thank you very much in advance for your contributions!! Please ensure you refer to our [Code of Conduct](https://github.com/hotosm/fAIr/blob/master/docs/Code-of-Conduct.md).
```

Comment:
Use relative file path as well here

Reply:
Hi Raj @kshitijrajsharma, thanks for your feedback and response🙂.
```
Please could you provide more detail on what you meant by *use relative file path as well here?*
```

Reply:
See [Relative links and image paths in README files](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files).